src/DataMiner/SLManagedAutomation/SubScriptOptions.cs

```diff
@@ -65,7 +65,7 @@ public class SubScriptOptions
 		public bool WaitWhenLocked { get; set; }
 
 		/// <summary>
-		/// Gets or sets a value indicating whether the script will return more detailed error information instead of throwing an exception. This is supported from DataMiner 10.3.0/10.2.7 onwards.
+		/// Gets or sets a value indicating whether the script will return more detailed error information instead of throwing an exception. These can be retrieved with method <see cref="SubScriptOptions.GetErrorMessages"/>. This is supported from DataMiner 10.3.0/10.2.7 onwards.
```

Comment:
```suggestion
		/// Gets or sets a value indicating whether the script will return more detailed error information instead of throwing an exception. These can be retrieved with method <see cref="SubScriptOptions.GetErrorMessages"/>. This is supported from DataMiner 10.3.0/10.2.7 onwards (RN 33306).
```